incident_viewer_demo/src/LineChart/LineChart.jsx

```diff
@@ -9,6 +9,7 @@ import { testThreshold } from '../chart_support/constants';
 
 import './LineChart.scss';
 import moment from 'moment';
+import { timingSafeEqual } from 'crypto';
```

Comment:
Commenting on older code again... the docstring below saying "Figure out what is needed to render the chart based on the props of the component" could have more detail. Does this mean that if incident data is not included in the props, then the code will function as normal without displaying any incident relevant lines/arrows?